### ISSUE TITLE
Add Integration tests for databases that require docker

### ISSUE THREAD

**Issue by Harlekuin** [Harlekuin/SimQLe#29]
- [ ] Sql Server
- [ ] MariaDB

**Comment by Harlekuin** [comment]
This script works for biolab/orange3 for SQL Server:

```bash
sudo docker run -e 'ACCEPT_EULA=Y' -e 'SA_PASSWORD=YourStrong!Passw0rd' \
   -p 1433:1433 -d microsoft/mssql-server-linux:2017-latest
```

**Comment by ZackBotkin** [comment]
Are you running most of the databases out of docker containers?

**Comment by Harlekuin** [comment]
Travis CI has options to turn on services such as certain databases so I've added integration tests for those first. See the integration tests folder for each of those. This issue is to track adding integration tests on Travis CI that require more manual set up

**Comment by Harlekuin** [comment]
I'm going to change this to be purely out of docker containers, so we can run the tests locally if required, and just use that as part of the Travis CI

**Comment by Harlekuin** [comment]
closed in #59
